boot/boot_serial/src/cbor_encode.h

```diff
@@ -55,10 +55,10 @@ bool bstrx_encode(cbor_state_t *state, const cbor_string_type_t *result);
 bool tstrx_encode(cbor_state_t *state, const cbor_string_type_t *result);
 
 #define tstrx_put(state, string) \
-	tstrx_encode(state, &(cbor_string_type_t){.value = (const uint8_t *)string, len = (sizeof(string) - 1)})
+	tstrx_encode(state, &(cbor_string_type_t){.value = (const uint8_t *)string, .len = (sizeof(string) - 1)})
```

Comment:
The file at the top contains sha of the commit in the cddl repo that the file comes from. I think that this should be updated, and corrected in all other cddl files.

Reply:
^^ I sent patch to cddl-gen as well, but i don't have any influence on when it will be merged.